egs/wsj/s5/steps/nnet3/multilingual/combine_egs.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+#
+# This script uses separate input egs directory for each language as input,
+# to generate egs.*.scp files in multilingual egs directory
+# where the scp line points to the original archive for each egs directory.
+# $megs/egs.*.scp is randomized w.r.t language id.
+#
+# Also this script generates egs.JOB.scp, output.JOB.scp and weight.JOB.scp,
+# where output file contains language-id for each example
+# and weight file contains weights for scaling output posterior
+# for each example w.r.t input language.
+#
+# Begin configuration section.
+cmd=run.pl
+minibatch_size=512      # multiple of actual minibatch size used during training.
```

Comment:
please remove the comment "multiple..." and say:
`this does not have to be the actual minibatch size; it is the number of consecutive
 egs that we take from each source, and it only affects the locality of disk access.`